include/iocore/net/ConnectionTracker.h

```diff
@@ -43,10 +44,11 @@
 #include "iocore/net/SessionSharingAPIEnums.h"
 
 /**
- * Singleton class to keep track of the number of outbound connections.
+ * Singleton class to keep track of the number of inbound and outbound connections.
  *
  * Outbound connections are divided in to equivalence classes (called "groups" here) based on the
- * session matching setting. Tracking data is stored for each group.
+ * session matching setting. Tracking data is stored for each group. Inbound connections always
```

Comment:
Maybe something more like "connections are divided into equivalence classes ... For outbound connections groups will vary based on session matching. For inbound a group is always based on the remote IP address".

Reply:
Yeah, that sounds good.